### DISCUSSION TITLE
GameSwitcher freezes after connecting to wifi

### DISCUSSION THREAD

**Discussion by whitehollow** [OnionUI/Onion#1282] [Support]
Hi dev team, can we please fix this issue where after connecting to wifi on miyoo mini plus and click the menu button to start GameSwitcher it goes to black screen?

If i click the power button for it to go to standby and then click again i can see the screenshot of the last played game but you cannot do anything afterwards.

The only workaround is to restart the device.

Thank you

**Comment by XK9274** [comment]
> Hi dev team, can we please fix this issue where after connecting to wifi on miyoo mini plus and click the menu button to start GameSwitcher it goes to black screen?
> 
> If i click the power button for it to go to standby and then click again i can see the screenshot of the last played game but you cannot do anything afterwards.
> 
> The only workaround is to restart the device.
> 
> Thank you

I've put up some findings in the issue,  we will continue the conversation there.
